File: homework10/Q1/q1.js
// 1. Write the following programs:
//  - Write a JavaScript program that will loop through array a, and capitalizes every third word.


var a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"];


{
    var j;
for (j=0;j<a.length; j++){
var word1=a[j];
var word2="";
if ((j+1)%3===0){
    var i;
  for (i = 0; i< word1.length; i++){
var aChar = word1.charCodeAt(i);
    if (65 <= aChar && aChar<=90){
    aChar =aChar + 32;
    }
    word2=word2+word1.charAt(i).toUpperCase();
 }
console.log ( word2);
 }   
}}

File: homework10/Q1/q1.rb
var a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"];

j=0
while j<a.length
 word1=a[j]
 word2=""
if ((j+1)%3==0)
    i = 0
    while i< word1.length
 aChar = word1.charCodeAt(i);
    if (65 <= aChar && aChar<=90)
    aChar =aChar + 32
end
    word2=word2+word1.charAt(i).toUpperCase();
 end
 i=i+1
 end
puts ( word2);
 end
j=j+1
end

